specification/elastic/resource-manager/Microsoft.Elastic/stable/2023-06-01/elastic.json

```diff
@@ -0,0 +1,2510 @@
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AllTrafficFilters_list": {
+            "$ref": "./examples/AllTrafficFilters_list.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/listAssociatedTrafficFilters": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Get the list of all associated traffic filters for the given deployment.",
+        "operationId": "listAssociatedTrafficFilters_list",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ElasticTrafficFilterResponse"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "listAssociatedTrafficFilters_list": {
+            "$ref": "./examples/AssociatedFiltersForDeployment_list.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/createAndAssociateIPFilter": {
+      "post": {
+        "tags": [
+          "TrafficFilter"
+        ],
+        "summary": "Create and Associate IP traffic filter for the given deployment.",
+        "operationId": "createAndAssociateIPFilter_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "ips",
+            "description": "List of ips",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "description": "Name of the traffic filter",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
```

Comment:
Add response schema for 200 response.

Reply:
@visingla-ms 
As discussed offline, we're just creating a stable version for the preview API versions which are already available and no new changes are being pushed here.
PFB the previous PRs:
https://github.com/Azure/azure-rest-api-specs/pull/22638
https://github.com/Azure/azure-rest-api-specs/pull/23343
https://github.com/Azure/azure-rest-api-specs/pull/23602
https://github.com/Azure/azure-rest-api-specs/pull/23650
https://github.com/Azure/azure-rest-api-specs/pull/24031